cluster-service/cspr/namespace-cleaner.yaml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Now we have a new OCP Template, but is it being deployed? I only see the addition of the new template file

Reply:
yes, for now, I have added the template only for you to take a look, I will make the changes to have it deployed also.